### PR TITLE
Patch for router

### PR DESCRIPTION
**Pull request by Natim**
Let's authenticate on the right req.originalUrl if possible.

### CODE REVIEW

File: lib/hawk.js [hunk 1]
```diff
@@ -95,12 +95,19 @@ function getMiddleware(options) {
 
   if (sendError === undefined) {
     sendError = function(res, status, payload) {
-      res.json(status, payload);
+      res.status(status).json(payload);
```
Comment (almet) [inline lib/hawk.js:98]: which version of express introduced that, are we becoming incompatible with releases previous this change?

Comment (Natim) [inline lib/hawk.js:98]: res.status().json() seems to work with express 3.x, It is the reason why I am now testing for both express 3x and express 4x here: https://github.com/mozilla-services/express-hawkauth/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R6

But maybe we should add a specific test for it.

**Comment by almet** [comment]
Looks good to me, I want to double check about the forward compat :)

**Comment by Natim** [comment]
I have added a test to make sure it works.

**Comment by almet** [comment]
Cool, thanks!

**Pull request merged**